src/components/storagePools/createStoragePoolDialog.jsx

```diff
@@ -151,6 +151,7 @@ const StoragePoolSourceRow = ({ onValueChanged, dialogValues }) => {
     let validationState;
     let placeholder;
     const diskPoolSourceFormatTypes = ['dos', 'dvh', 'gpt', 'mac'];
+    const diskPoolSourceFsTypes = ['auto', 'ext2', 'ext3', 'ext4', 'xfs'];
```

Comment:
Could you please add a comment with a reference URL which defines this list? This is prone to getting out of date, so our future selves in a few years will appreciate :grin:

Reply:
I added it according to the advice.

Reply:
Thanks! The comment should go above the line though, and the URL is enough.

Reply:
Thank you, I changed the location.